manifests/r/Remeha/ServiceTool/7.0.53.0/Remeha.ServiceTool.installer.yaml

```diff
@@ -0,0 +1,17 @@
+# Created with YamlCreate.ps1 v2.2.12 $debug=AUSU.CRLF.5-1-19041-3693.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Remeha.ServiceTool
```

Comment:
Should this package be identified as `Remeha.Recom`?

![image](https://github.com/microsoft/winget-pkgs/assets/114952565/457ee036-e4f5-4132-ac15-34ef63e72e17)

> Description: Recom is a software program developed by Remeha B.V. It offers the possibility to communicate directly to Remeha boilers/appliances.

Reply:
I don't think so, but `recom` is added as a moniker anyways, so it should still appear in search